docs/vm-images.md

```diff
@@ -207,10 +207,10 @@ description of our support for Windows images.
 <tr>
   <th>Startup script</th>
   <th></th>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
-  <td><a href="../tools/validate_configs/os_compatibility_tests/slurm-startup.yaml">✓</a></td>
+  <td><a href="../community/examples/hpc-slurm-ramble-gromacs.yaml">✓</a></td>
+  <td>✓</td>
+  <td>✓</td>
```

Comment:
Same here as #2327 
We will want to update this once we migrate `tools/validate_configs/test_configs/slurm-gcp-v5-startup-scripts.yaml`

Reply:
Added this PR as well in the canonical bug assigned to me. Will update the doc once we have v6 examples.